src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -14,9 +14,13 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { push, Push } from 'connected-react-router';
```

Comment:
Do we know whether we prefer a `push` or a `replace` here? I.e. when user clicks on the `back` button in the browser, do we want to cycle through the tabs in the gene function panel, or do we want to transport them to a different screen?

Reply:
I am not sure what we would prefer. I'll check this with Andrea.

Reply:
I would prefer cycling through the tabs as the user still can get back to a different screen where they came from if needed.